crates/goose-server/src/tunnel/config.rs

```diff
@@ -0,0 +1,49 @@
+use super::TunnelConfig;
```

Comment:
I don't think this module adds all that much. we create a layer between the config object we already use but can't we just call everything directly? now when you change the auto start we also write the secret etc.

just read and write it when we need it, is what I would think

Reply:
yeah I think this is a remnant from porting it from ts.